src/devices/src/virtio/queue.rs

```diff
@@ -93,7 +93,7 @@ impl<'a> DescriptorChain<'a> {
             Ok(ret) => ret,
             Err(err) => {
                 // TODO log address
-                error!("Failed to read virtio descriptor from memory: {}", err);
+                error!("Failed to read virtio descriptor from memory at address {desc_head:#x}: {err}");
```

Comment:
[`GuestAddress`](https://github.com/rust-vmm/vm-memory/blob/1bdb2be7c4dfd16694251c90f6799312138cdc20/src/guest_memory.rs#L129) does not implement `LowerHex`, you'll have to grab the `u64` it contains with `desc_head.0`, and then format that